hydro-knowrob/hydro-knowrob-daemon/Dockerfile

```diff
@@ -28,4 +28,6 @@ ENV OPEN_EASE_APPLICATION true
 # build the catkin workspace
 RUN /opt/ros/hydro/bin/catkin_make
 
+VOLUME /home/ros/summary_data
```

Comment:
Did this change get in by accident? We are mounting summary_data into the user containers here https://github.com/knowrob/docker/blob/master/hydro-knowrob/hydro-knowrob-base/Dockerfile#L51 and in the knowrob container here https://github.com/knowrob/docker/blob/master/webapps/knowrob/Dockerfile#L8, with summary_data being created here https://github.com/knowrob/docker/blob/master/scripts/start-webrob#L62.
With this declaration the user container is providing the volume /home/ros/summary_data, which should be done only by the summary_data data container.